client/src/main/java/au/org/consumerdatastandards/client/api/BankingPayeesAPI.java

```diff
@@ -38,7 +38,7 @@
      * http.response.details
      * <table summary="Response Details" border="1">
      *   <tr><td> Status Code </td><td> Description </td><td> Response Headers </td></tr>
-     *   <tr><td> ResponseCode.OK </td><td> Success </td><td>  -  </td></tr>
+     *   <tr><td> 200 </td><td> Success </td><td>  * x-v - The [version](#response-headers) of the API end point that the data holder has responded with. <br>  * x-fapi-interaction-id - An [RFC4122](https://tools.ietf.org/html/rfc4122) UUID used as a correlation id. If provided, the data holder must play back this value in the x-fapi-interaction-id response header. If not provided a [RFC4122] UUID value is required to be provided in the response header to track the interaction. <br>  </td></tr>
```

Comment:
Something is seriously wrong with this line here and in similar classes.

Reply:
fixed